automation_tools/__init__.py

```diff
@@ -2942,18 +2942,19 @@ def katello_installer(debug=False, distribution=None, verbose=True,
     ))
 
 
-def katello_service(action, exclude=None):
+def katello_service(action, sat_version='', exclude=None):
     """Run katello-service
 
     :param str action: One of stop, start, restart, status.
     :param list exclude: A list of services to skip
 
     """
-    if exclude is None:
-        exclude = ''
+    exclude = '--exclude {0}'.format(','.join(exclude)) if exclude else ''
+    if (sat_version.split('.')) > ('6', '7'):  # sat_version > '6.7'
```

Comment:
Not sure, how it will compare upstream-nightly?

Reply:
It sorts it correctly (i just had an error in the statement):
```python

In [26]: tuple('foreman_nightly'.split('.')) > ('6', '7')
Out[26]: True

In [27]: tuple('aaaa'.split('.')) > ('6', '7')
Out[27]: True
```